packages/trader/src/Modules/Trading/Containers/trade-chart.tsx

```diff
@@ -0,0 +1,200 @@
+        isHighestLowestMarkerEnabled: false, // TODO: Pending UI,
+        language: current_language.toLowerCase(),
+        position: is_chart_layout_default ? 'bottom' : 'left',
+        theme: is_dark_mode_on ? 'dark' : 'light',
+        ...(is_accumulator ? { whitespace: 190, minimumLeftBars: is_mobile ? 3 : undefined } : {}),
+    };
+
+    const { current_spot, current_spot_time } = accumulator_barriers_data || {};
+
+    const getBottomWidgets = React.useCallback(
+        ({ digits, tick }: TBottomWidgetsParams) => (
+            <ChartBottomWidgets digits={digits} tick={tick} show_accumulators_stats={is_accumulator} />
+        ),
+        [is_accumulator]
+    );
+
+    const getMarketsOrder = (active_symbols: ActiveSymbols): string[] => {
+        const synthetic_index = 'synthetic_index';
+
+        const has_synthetic_index = !!active_symbols.find(s => s.market === synthetic_index);
+        return active_symbols
+            .slice()
+            .sort((a, b) => (a.display_name < b.display_name ? -1 : 1))
+            .map(s => s.market)
+            .reduce(
+                (arr, market) => {
+                    if (arr.indexOf(market) === -1) arr.push(market);
+                    return arr;
+                },
+                has_synthetic_index ? [synthetic_index] : []
+            );
+    };
+
+    const barriers = main_barrier ? [main_barrier, ...extra_barriers] : extra_barriers;
+
+    // max ticks to display for mobile view for tick chart
+    const max_ticks = granularity === 0 ? 8 : 24;
+
+    if (!symbol || active_symbols.length === 0) return null;
+
+    return (
+        <SmartChartSwitcher
+            allowTickChartTypeOnly={show_digits_stats || is_accumulator}
+            barriers={barriers}
+            bottomWidgets={
+                (is_accumulator || show_digits_stats) && isDesktop() ? getBottomWidgets : props.bottomWidgets
+            }
+            chartControlsWidgets={null}
+            chartData={{
+                activeSymbols: JSON.parse(JSON.stringify(active_symbols)),
+            }}
+            chartStatusListener={(v: boolean) => setChartStatus(!v)}
+            chartType={chart_type}
+            clearChart={false}
+            contracts_array={markers_array}
+            crosshair={is_mobile ? 0 : undefined}
+            crosshairTooltipLeftAllow={560}
+            feedCall={{
+                activeSymbols: false,
+            }}
+            enabledNavigationWidget={isDesktop()}
+            enabledChartFooter={false}
+            getMarketsOrder={getMarketsOrder}
+            granularity={show_digits_stats || is_accumulator ? 0 : Number(granularity)}
+            hasAlternativeSource={has_alternative_source}
+            id='trade'
+            importedLayout={chart_layout}
+            initialData={{
+                activeSymbols: JSON.parse(JSON.stringify(active_symbols)),
+            }}
+            isConnectionOpened={is_socket_opened}
+            isLive={true}
```

Comment:
`isLive` can be without `true`?

Reply:
when flutter-charts was added, it was done this way. but surely, can omit

Reply:
No, Smartcharts V2 needs isLive 😅 
May be can fix it later in Smartcharts.